suites/reef/rados/tier-2_rados_test-bugfixes.yaml

```diff
@@ -168,3 +168,15 @@ tests:
         slow-osd-heartbeat: true
       polarion-id: CEPH-83590688
       desc: Generate Slow OSD heartbeats by inducing network delay
+
```

Comment:
Is the test fix added only in reef?  If that's the case, let's check if a backport bug needs to be created from QA side.

Reply:
Backport bug is raised for Quincy- https://bugzilla.redhat.com/show_bug.cgi?id=2293908